### ISSUE TITLE
Remove Squeeze support

### ISSUE THREAD

**Issue by myhro** [andsens/bootstrap-vz#280]
As yesterday (2016/02/06), `squeeze` [is not supported anymore][1], as it has completed five years since its launch and LTS period has ended.

Any tasks/workarounds/etc. created to specifically support this release should be removed from the code base and documentation.

[1]: https://www.debian.org/releases/squeeze/

**Comment by andsens** [comment]
> Any tasks/workarounds/etc. created to specifically support this release should be removed from the code base and documentation.

Agreed. Though we should tag the commit before that, just so people can get the latest version that bootstraps squeeze.

**Comment by myhro** [comment]
Anders,

Is it okay if I tag 26d09d3 as `0.9.9-squeeze`? Because 790f2d6 already removes `squeeze` mentions from the documentation about supported builds.

**Comment by andsens** [comment]
Yup, that's fine. Go right ahead.

**Comment by myhro** [comment]
Done: https://github.com/andsens/bootstrap-vz/tree/v0.9.9-squeeze

P.s.: there was a [`before-wip-integration-testing-rebase`][1] tag which was pushed together with the intended one. Should we delete it?

[1]: https://github.com/andsens/bootstrap-vz/tree/before-wip-integration-testing-rebase

**Comment by myhro** [comment]
There's an [ongoing discussion][1] about when Squeeze LTS support will really end. It maybe done at the [end of the month][2], but the kernel from 2.6 series had [its support finished already][3].

[1]: https://lists.debian.org/debian-lts/2016/02/msg00036.html
[2]: https://lists.debian.org/debian-lts/2016/02/msg00044.html
[3]: https://lists.debian.org/debian-lts/2016/02/msg00045.html
